__tests__/app.test.js

```diff
@@ -80,4 +80,25 @@ describe("/api/app", () => {
         expect(body.msg).toBe("No review found for review_id: 10000");
       });
   });
+  test("Reviews should have properties of title, designer, review_body, review_id, review_img_url, votes, owner, created_at, category, comment_count", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.reviews).toHaveLength(13);
+        expect(body.reviews).toBeSorted("created_at", { descending: true });
```

Comment:
It might be that this works as is, but looking at the docs for `jest-sorted` I think you'd want to use the matcher ._toBeSortedBy_ rather than _.toBeSorted_  when giving it a key to sort by

Reply:
ok